libs/core/src/components/pds-popover/docs/pds-popover.mdx

```diff
@@ -0,0 +1,101 @@
+
+## Guidelines
+### Usage
+- Use popovers for complex information that might overwhelm the main content.
+- Consider using a tooltip for simpler explanations or brief labels.
+
+#### When not to use
+- Don't use popovers for critical information that must be readily available.
+
+## Properties
+
+<DocArgsTable componentName="pds-popover" docSource={components} />
+
+## Variants
+### Default
+The popover's appearance and behavior are primarily controlled by two slots:
+1. **`trigger`:** This slot is used to define the element that triggers the popover. It can be any HTML element, such as a button or link.
+2. **`content`:** This slot is used to define the content of the popover. It can contain any HTML elements, including text, images, forms, or other components.
+
+
+<DocCanvas client:only
+  mdxSource={{
+    react: `
+      <PdsPopover placement="bottom-start">
+        <div slot="trigger">
+          <PdsButton variant="secondary">Popover</PdsButton>
+        </div>
+        <div slot="content">
+          <p>This is a Popover</p>
+        </div>
+      </PdsPopover>
+    `,
+    webComponent: `
+      <pds-popover placement="bottom-start">
+        <div slot="trigger">
+          <pds-button variant="secondary">Popover</pds-button>
+        </div>
+        <div slot="content">
+          <p>This is a Popover</p>
+        </div>
+      </pds-popover>
+    `
+}}>
+  <pds-popover placement="bottom-start">
+    <div slot="trigger">
+        <pds-button variant="secondary">Popover</pds-button>
+    </div>
+    <div slot="content">
+        <p>This is a Popover</p>
+    </div>
+  </pds-popover>
+</DocCanvas>
+
+### Positioning
+Using the placement prop, you can control where the popover appears relative to the trigger.
+
+### Arrow
+The popover includes an arrow by default, but you can hide it with the `has-arrow` prop.
+
+<DocCanvas client:only
+  mdxSource={{
+    react: `
+      <PdsPopover placement="bottom-start" hasArrow="false">
+        <div slot="trigger">
+          <PdsButton variant="secondary">Popover</PdsButton>
+        </div>
+        <div slot="content">
+          <p>This is a Popover</p>
+        </div>
+      </PdsPopover>
+    `,
+    webComponent: `
+      <pds-popover placement="bottom-start" has-arrow="false">
+        <div slot="trigger">
+          <pds-button variant="secondary">Popover</pds-button>
+        </div>
+        <div slot="content">
+          <p>This is a Popover</p>
+        </div>
+      </pds-popover>
+    `
+}}>
+  <pds-popover placement="bottom-start" has-arrow="false">
+    <div slot="trigger">
+        <pds-button variant="secondary">Popover</pds-button>
+    </div>
+    <div slot="content">
+        <p>This is a Popover</p>
+    </div>
+  </pds-popover>
+</DocCanvas>
+
+## Additional Resources
+
+- [W3 Aria Menu Button Pattern](https://www.w3.org/WAI/ARIA/apg/patterns/menu-button/examples/menu-button-links/)
```

Comment:
Could we review this link and provide a little bit of context as to why this is relevant to the popover? Thanks!